File: C/fizzBuzz.c
#include <stdio.h>

int xx;

int main(void){
 for(xx=1;xx<=50;xx++){
        if (xx%3 == 0 && xx%5 == 0){
            printf("FizzBuzz\n");
        }else if (xx%3 == 0){
            printf("Fizz\n");
        }else if (xx%5 == 0){
            printf("Buzz\n");
        }else{
            printf("%d\n", xx);
        }
 }
    return 0;
}

File: Python/fizzBuzz.py
x = 0

num = 50




while x < num:
    x += 1
    if x%3 == 0 and x%5 == 0 and x != 0:
        print("FizzBuzz")
    elif x%3 == 0 and x != 0:
        print("Fizz")
    elif x%5 == 0 and x != 0:
        print("Buzz")
    else:
        print(x)